src/main/resources/db/migration/V2_0__init.sql

```diff
@@ -1,25 +1,30 @@
 create table if not exists appuser
 (
-    id              bigserial   primary key,
-    id_telegram     bigint not null,
-    city            varchar(255),
-    language        varchar(255),
-    name            varchar(255),
-    phone           varchar(255),
-    time_created    timestamp
+    id           bigserial
+        primary key,
+    city         varchar(255),
+    created_time timestamp,
+    language     varchar(255),
+    name         varchar(255),
+    phone        varchar(255),
```

Comment:
```suggestion
    phone        varchar(20),
```
I think 20 will be enough